### ISSUE TITLE
Something went wrong! Without any further hints/instructions.

### ISSUE THREAD

**Issue by MNF** [integrations/microsoft-teams#88]
Our admin recently installed Teams  integration on our organization GitHub
And received message

> Hi , you've successfully installed GitHub app on this team.
> To subscribe to a repository, use the following command:
> @github subscribe owner/repository

I’ve installed app on my desktop Teams and tried 

[13:16] Michael Freidgeim
subscribe MyOrganisation/MyRepository
​
[13:16] GitHub (Preview)
Connect to your GitHub account before subscribing to a repository
​
[13:16] GitHub (Preview)
Activity Id: 2a22c212-4f81-4558-9b8f-4675f507ce7a. Something went wrong!

The message is not very informative. Can the message include some instructions what can I do with this activity Id ?
As it is still preview, you may consider to show details of exception ( which is not acceptable for final product, but may give some hints to preview users)

**Comment by gauravsaralMs** [comment]
@MNF thanks for reporting the issue.. yes we will look into how to enhance this experience.. in the mean time can you let us know if you are able to run the 
`signin` command and `subscribe yourOrganization/YourRepository` separately ?

**Comment by schoeps** [comment]
Maybe completely unrelated but I get the same error message (with a different Activity Id) when trying to connect my GitHub account.

**Comment by gauravsaralMs** [comment]
@schoeps : yes this is a generic error handling.. basically when our system encounters something which it does not expect then we use this route.

can you share if you ran into this while running `signin` or something else?

**Comment by schoeps** [comment]
Yes, the error pops up after clicking `signin` using the correct password and entering the two factor code. I am using the latest Teams client with the github (preview) app for macOS on Apple Silicon. I uninstalled the github (legacy) app.

**Comment by gauravsaralMs** [comment]
@schoeps . can you please share the error message you are getting? 
that activity id will help us track and root cause the issue effectively.

**Comment by schoeps** [comment]
> @schoeps . can you please share the error message you are getting?
> that activity id will help us track and root cause the issue effectively.

@gauravsaralMs Sure! Thanks. `Activity Id: 96e493d6-7d6d-42ba-82cd-f23924b3c2af. Something went wrong!`

**Comment by gauravsaralMs** [comment]
Thanks for reporting this @schoeps , one of the partner services is rejecting the calls here.. 
but we have r[eached out to the partner team](https://github.com/microsoft/botbuilder-js/issues/3298) to check on what can be the root cause..

we will keep this thread updated.

cc: @snankani @ashokirla

**Comment by schoeps** [comment]
@gauravsaralMs we are running an educational Teams installation in Germany which is hosted on German servers (due to German/European data protection laws) as far as I know. Maybe this helps.

**Comment by gauravsaralMs** [comment]
thanks @schoeps , we will communicate the same to partner teams.. looking at the error message I think this is of help

**Comment by nspyke** [comment]
Hi @gauravsaralMs 
I'm seeing the same error after trying to sign in.
```
[9:41 AM] GitHub (Preview)
    Activity Id: 690b17aa-a9da-44dd-a19a-9205bf8d8351. Something went wrong!
​[9:41 AM] GitHub (Preview)
    Activity Id: c124b9ec-b60e-4912-ac87-f5b1c7ab9962. Something went wrong!
```

**Comment by MattiPuolitaivalGKGAB** [comment]
Any updates to this?

I'm receiving the same error. Got it on my old Win10 laptop and now also with the new M1 MBP I recently received.

**Comment by anuraag016** [comment]
@MattiPuolitaivalGKGAB, as mentioned in this comment https://github.com/integrations/microsoft-teams/issues/88#issuecomment-777258839, one of the partner services is rejecting the calls here. 

We had opened an [issue](https://github.com/microsoft/botbuilder-js/issues/3298) under the partner team (which turned out to be a duplicate of this [issue](https://github.com/microsoft/botbuilder-js/issues/3280)), the work to fix this issue is still pending. 
We will update once the issue is resolved.

**Comment by thxkiwi** [comment]
Still seeing this 2 years later. Any update?

**Comment by Peter-Rasborg** [comment]
Best and correct installations
